save-backend/src/main/kotlin/org/cqfn/save/backend/entities/Project.kt

```diff
@@ -0,0 +1,14 @@
+package org.cqfn.save.backend.entities
+
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+
+@Entity
+class Project(
```

Comment:
Also, we initially planned to store these classes in `save-common` to access them from both frontend and backend. Did you investigate this possibility?